tests/examples/examples.test.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2024 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { exec } from "child_process";
+import { glob } from "glob";
+import { promisify } from "util";
+
+const execAsync = promisify(exec);
+const pattern = process.env.EXAMPLE_NAME || `./examples/**/*.ts`;
+
+const exclude: string[] = []; // list of examples that are excluded
```

Comment:
How can I address these?  The ENVs for the test(examples.test.ts) are passed into the examples.  So if the necessary ENVs are defined in the github action, it should work.  So I assume that some examples never get the necessary ENVs.  Do you want to update the examples not to fail so that they don't be in the exclude list?

Reply:
>  So I assume that some examples never get the necessary ENVs.
It would be great if one could run that locally. Only some have all environments set, so it makes sense to ignore those examples that cannot be run. How? You have to manually look at all examples and find out which need which ENVs. I can do that later if you don't want to.

Reply:
I'll do that.  I wanted to know how you want to address these.  Thanks!

Reply:
I believe that some tests that wait for user input also fail.  Do you want to exclude those, too?

Reply:
Good point. You can pass the prompt to them like this.

```
yarn start -- ${example} <<< "Hello world!"
```